changelog/change_add_rubyjard_debugger_calls_to.md

```diff
@@ -0,0 +1 @@
+* [#10034](https://github.com/rubocop/rubocop/pull/10034): Add `RubyJard` debugger calls to Lint/Debugger/DebuggerMethods. ([@DanielVartanov][https://github.com/DanielVartanov])
```

Comment:
Can you update below?

```suggestion
* [#10034](https://github.com/rubocop/rubocop/pull/10034): Add `RubyJard` debugger calls to Lint/Debugger/DebuggerMethods. ([@DanielVartanov][])
```

Reply:
Already did, sorry about that, I misread the instruction in CONTRIBUTION